### DISCUSSION TITLE
EN 11.7: What is meant with units of measurement?

### DISCUSSION THREAD

**Discussion by sander-nl** [Nordic-Accessibility-Community-Group/en-301-549-resources-and-eaa-monitoring#1] [Q&A]
As discussed in [“EN 301 549 vs WCAG – Gaps and Bridges” Study Group](https://www.w3.org/community/nordic-accessibility/2024/11/07/en-301-549-vs-wcag-gaps-and-bridges-study-group/) on 2024-11-07:

[Clause 11.7 in EN 301 549](https://accessible.canada.ca/en-301-549-accessibility-requirements-ict-products-and-services-11-software#_Toc66969652) states: 

> Where software is not designed to be isolated from its platform, and provides a user interface, that user interface shall follow the values of the user preferences for platform settings for: units of measurement, (...) except where they are overridden by the user.

What is meant with units of measurement? Is this also about inches vs cm and different currencies?

**Comment by 2biazdk** [comment]
I agree that it seems quite broad and open to interpretation. It could be anything from time formats (12-hour vs 24-hour) to temperature units (Celsius vs Fahrenheit), calendar formats (week starting on Sunday vs Monday) or numbering conventions (e.g., commas vs periods as decimal separators).

It would have been helpful if EN 301 549 provided specific examples or clearer definitions to avoid confusion and make implementation more straightforward.

Simply giving some examples might be helpful. E.g.,:
> "Where software is not designed to be isolated from its platform, and provides a user interface, that user interface shall follow the values of the user preferences for platform settings for: units of measurement (e.g., distances, weights, temperatures, monetary values, and date/time formats), colour, contrast, font type, font size, and focus cursor, except where they are overridden by the user."

But perhaps it is deliberately left ambiguous, to not leave out any use-cases.

**Reply by thniels** [reply]
> But perhaps it is deliberately left ambiguous, to not leave out any use-cases.

That would make sense, seeing that every domain has its specificities. Not just in technical jargon but certainly also in user interface representation.

**Comment by sander-nl** [comment]
In Sweden, monitoring body DIGG has assessed in april 2024 that it only concerns temperature and distance and that it does not apply to measurements in running texts. 

Sources in Swedish: [t12t Meetup Global Accessibility Awareness Day med Whitespace (starting around 24:00)](https://www.youtube.com/live/lYOkMFrmjUc?feature=shared&t=1433) and [Webbriktlinje 11.7 (partly)](https://www.digg.se/webbriktlinjer/alla-webbriktlinjer/respektera-anvandarens-installningar)
